src/Contracts/Models/Artifact.cs

```diff
@@ -1,13 +1,16 @@
-﻿using Newtonsoft.Json;
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using Microsoft.Extensions.Configuration;
 
 namespace Monai.Deploy.WorkflowManager.Contracts.Models
 {
     public class Artifact
     {
-        [JsonProperty(PropertyName = "name")]
+        [ConfigurationKeyName("name")]
```

Comment:
For models, this can stay as `JsonProperty` unless you need to access them through `IOptions`

Reply:
Ahh my bad, misunderstood, changed all these now.